### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by RajaVamsi11**
**Summary**
Fixed some minor and grammatical issues in contributing.md file

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@luckyman20 Can you please review this PR.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -14,25 +14,25 @@ There are two steps, you have to follow, when you are building the mobile-wallet
...
 
-or if you have ssh set up you can use that url instead:
+or if you have ssh set up you can use that URL instead:
 
 `git remote add upstream git@github.com:openMF/mobile-wallet.git`
 
 ## Commit Style Guide
 
- Mifos Mobile Wallet Android application have set of [Commit Style Guidelines](https://github.com/openMF/mobile-wallet/wiki/Commit-style-guide). we strictly following this guideline to track the every change, any bug fixes, any enhancement and any new feature addition. we are suggesting you, please follow these guidelines to help us managing every commit.
+ Mifos Mobile Wallet Android application has set of [Commit Style Guidelines](https://github.com/openMF/mobile-wallet/wiki/Commit-style-guide). we strictly following this guideline to track every change, any bug fixes, any enhancement, and any new feature addition. we are suggesting you, please follow these guidelines to help us managing every commit.
```
Comment (luckyman20) [inline .github/CONTRIBUTING.md]:
```suggestion
 Mifos Mobile Wallet Android application have a set of [Commit Style Guidelines](https://github.com/openMF/mobile-wallet/wiki/Commit-style-guide). We strictly follow this guideline to track every change, any bug fix, any enhancement, and any new feature addition. We are suggesting you to please follow these guidelines to help us managing every commit.
```

**Comment by RajaVamsi11** [comment]
@luckyman20 Changes done.

**Comment by luckyman20** [comment]
@RajaVamsi11 Please squash the commits.

**Pull request merged**